content/en/blog/2023/istioday-kubecon-eu/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Join us for IstioDay at KubeCon Europe 2023!"
+description: The first ever IstioDay at KubeCon will take place in April.
+publishdate: 2023-01-26
+attribution: "Istio Steering Committee"
+keywords: [IstioCon,Istio,conference]
+---
+
+IstioDay Europe 2023, set for April 18, will be the first ever conference for Istio, the industry’s [most popular service mesh](https://www.cncf.io/wp-content/uploads/2020/11/CNCF_Survey_Report_2020.pdf), collocated with the KubeCon. Istio, which is currently [a graduating project at CNCF](https://www.cncf.io/projects/istio/) will mark its presence at the KubeCon in 2023, connecting community members across the globe with Istio’s ecosystem. Visit the [conference website](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/istio-day/) for all the information related to the event.
+
+{{< image width="75%"
+    link="./istioday-eu-logo.png"
+    alt="IstioDay logo"
+    >}}
+
+IstioDay is the perfect opportunity to showcase the lessons learned from running Istio in production, hands-on experiences from the Istio community, and will feature maintainers from across the Istio ecosystem. IstioDay would focus on sharing real world examples, case studies, and success stories that can inspire newcomers to use Istio in production. The content will range from introductory to advanced levels, split into four main topic tracks:
+
+* **New Features:** New standards, new extensions and how to address problems that are interesting to tackle
+* **Case Studies:** Observability, traceability, and other things built on top of Istio.
+* **Istio Recipes:** How Istio works, with deep-dives into performance, cost, and multi-cloud environments.
+* **Project Updates:** The evolution of Istio, and the latest updates from the project maintainers.
+
+At this time, we encourage Istio users, developers, partners, and advocates to [submit a session proposal through the conference’s CFP portal](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/cfp-colocated-events/). IstioDay offers a mix of technical talks, lightning talks, workshops, and roadmap sessions. Choose from the following formats to submit a session proposal for IstioDay:
+
+* **Presentation:** 25 minutes, 1 or 2 speaker(s) presenting a topic
+* **Panel Discussion:** 35 minutes of discussion among 3 to 5 speakers
+* **Lighting Talk:** A brief 5-minute presentation, maximum of 1 speaker
+
+At KubeCon Europe, Istio community also has an interactive social hour to take the load off and mesh with the Istio community, vendors, and maintainers. IstioDay is one of the `KubeCon + CloudNativeCon Europe CNCF-hosted Co-located Events`. In-person attendees will now have the option to register for an `All-Access In-Person KubeCon + CloudNativeCon pass` that will include entry to all `CNCF-hosted co-located events + KubeCon + CloudNativeCon`.
```

Comment:
Can we provide a link to the social?

Reply:
I think we do not have a link at the moment. Should I remove this mention?